assets/js/components/data-table.js

```diff
@@ -91,13 +92,17 @@ export const getDataTableFromData = ( data, headers, options ) => {
 	} );
 
 	return (
-		<div className={ `googlesitekit-table${ ( options && options.disableListMode ? '' : ' googlesitekit-table--with-list' ) }` }>
-			<table
-				className={ `
-					googlesitekit-table__wrapper
-					googlesitekit-table__wrapper--${ data && data[ 0 ] ? data[ 0 ].length : 1 }-col
-				` }
-			>
+		<div className={ classnames(
+			'googlesitekit-table',
+			{ 'googlesitekit-table--with-list': ! options || ! options.disableListMode }
+		) }>
+			<table className={ classnames(
+				'googlesitekit-table__wrapper',
+				{
+					'googlesitekit-table__wrapper-1': ! data && ! data[ 0 ].length,
+					[ `googlesitekit-table__wrapper--${ data[ 0 ].length }-col` ]: data && data[ 0 ].length,
```

Comment:
This should check for `data[ 0 ]` as well:
```suggestion
					[ `googlesitekit-table__wrapper--${ data[ 0 ].length }-col` ]: data && data[ 0 ] && data[ 0 ].length,
```